node_modules/credscan-pkg/main.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const config = require('./config.json');
+const os = require('os');
+var version = 'core'
+var namespace = 'CredScanSrc'
+if (version === 'core') version = 'coreapp';
```

Comment:
what is the need for checking version

Reply:
as we are defining it in line 4 also do we need line 11 check?

Reply:
I'll remove it